query04.sql

```diff
@@ -0,0 +1,24 @@
+alter table septa_bus_shapes
+    add column the_geom geometry(Geometry, 32129);
+
+update septa_bus_shapes
+    set the_geom = ST_Transform(ST_SetSRID(st_makepoint(
+	               shape_pt_lon, shape_pt_lat),4326),32129);
+
+with septa_bus_lines as (
+	SELECT shape_id, 
+      ST_MakeLine(the_geom) As linepath
```

Comment:
This will likely work (because most GTFS feeds come with the route shape rows ordered appropriately), but not necessarily. I recommend explicitly using `ST_MakeLine(the_geom ORDER BY shape_pt_sequence)`.